templates/test/ci/prow-ci-version/patches/machine-deployment-ci-version.yaml

```diff
@@ -12,4 +12,4 @@ spec:
           publisher: cncf-upstream
           offer: capi
           sku: ubuntu-1804-gen1
-          version: latest
+          version: "latest"
```

Comment:
Did this need to be quoted?

Reply:
I'm not sure, should I update this?

Reply:
<shrug> No I was just curious, it seems like a no-op change and now I see where it comes from (it was at "latest" with quotes before #2616 merged).